src/main/java/seedu/jarvis/logic/commands/add/AddCommand.java

```diff
@@ -21,7 +21,8 @@
             + "add -d DESCRIPTION d/YYYY-MM-DD t/HH:MM: Add deadline with DESCRIPTION at specified date and time\n";
 
     public static final String MESSAGE_MISSING_DESCRIPTION = "Please include the DESCRIPTION";
-    public static final String MESSAGE_MISSING_DATE = "Please include DATE and TIME d/YYYY-MM-DD t/HH:MM";
```

Comment:
I believe the TaskCommandParser needs MESSAGE_MISSING_DATE instead of MESSAGE_MISSING_INFO because Task dont have NAME variable.